src/components/snapshots/useSnapshot.ts

```diff
@@ -155,8 +155,10 @@ const useSnapshot: SnapshotOperation = async ({
     delay(1500)
 
     // Migrate database to latest version
-    console.log('Migrating database (if required)...)')
-    await migrateData()
+    if (options.shouldReInitialise) {
+      console.log('Migrating database (if required)...)')
+      await migrateData()
+    }
```

Comment:
Need to make sure migration *doesn't* run for template imports, otherwise the database will be temporarily non-functional on a live system. (even if only for a few seconds)